packages/markdownviewer/style/index.css

```diff
@@ -1,35 +0,0 @@
-/*-----------------------------------------------------------------------------
-| Copyright (c) Jupyter Development Team.
-| Distributed under the terms of the Modified BSD License.
-|----------------------------------------------------------------------------*/
-
-:root {
-  --jp-private-markdownviewer-padding: 16px;
-}
-
-.jp-MarkdownViewer {
-  border-top: var(--jp-border-width) solid var(--jp-border-color2);
-  outline: none;
-  overflow: hidden;
-  margin-top: -1px;
-  display: flex;
-  flex-direction: column;
-}
-
-
-.jp-MarkdownViewer .jp-Toolbar {
```

Comment:
Does the new widget we are using for rendering MIME based documents have the micro toolbar at the top?